xdsl_torch/utils/export_program.py

```diff
@@ -0,0 +1,32 @@
+import torch
+from xdsl.dialects import func
+from xdsl.ir import SSAValue
+
+from xdsl_torch.dialects.torch_mapping import REVERSE_XDSL_TORCH_OPS
+
+
+def export_program(func_op: func.FuncOp) -> torch.fx.Graph:
+    # TODO: instead of a Graph object construct a full ExportedProgram
+    graph = torch.fx.Graph()
+
+    if any(map(lambda arg: arg.name_hint is None, func_op.args)):
```

Comment:
I would just recommend against map everywhere, my understanding is that they keep it around for compatibility reasons but it's strictly worse than comprehensions
```suggestion
    if any(arg.name_hint is None for arg in func_op.args):
```